### ISSUE TITLE
Android modifyEvent()

### ISSUE THREAD

**Issue by BennoStaub** [EddyVerbruggen/Calendar-PhoneGap-Plugin#527]
Hey there

Is there a possibility to modify events for android? So far the only option I see would be to find the event, delete it and create the new modified event, but as we cannot define all parameters during event creation we couldn't always create an identical copy. What is the work around here?

Thanks!

**Comment by BennoStaub** [comment]
push :)

**Comment by davidecampello** [comment]
Hi!! Have you found a solution?

**Comment by BennoStaub** [comment]
> Hi!! Have you found a solution?

Unfortunately not. We try to avoid modifying events, otherwise we delete the old one and create a new updated one...

**Comment by davidecampello** [comment]
yes, It's a big limitation.. because there can be a loss of informations deleting events. And maybe id change can generate problems.
At the end we need modifications only on ios so It's ok, but I've seen that [there is a fork](https://github.com/Curve-Tomorrow/Calendar-PhoneGap-Plugin/commit/dc8e1c0e2664d4e8f0f9690e8ed0196e2546941d) where (it have to be tested and possibly manually modified) the modifications on android are implemented.
There are also other plugins, for example https://github.com/Fir3st/capacitor-calendar if you use capacitor, but It have to be tested..
Thanks for your answer!
